gulp/hygiene.js

```diff
@@ -17,8 +17,8 @@ module.exports = (blueprint, gulp, plugins) => {
         return del(cleanDirs, { force: true });
     });
 
-    gulp.task("tslint", () => (
-        gulp.src(["*.js", "gulp/**/*.js", "packages/*/*.js"])
+    gulp.task("tslint-gulp", () => (
```

Comment:
we already have a task group called `tslint` for the packages which now lints the JS files in the . this task lints only the gulp code, which lives in a separate directory structure